projects/packages/forms/src/dashboard/about/index.tsx

```diff
@@ -196,7 +194,7 @@ const About = () => {
 					<Details summary={ __( 'Is Jetpack Forms GDPR compliant?', 'jetpack-forms' ) }>
 						{ createInterpolateElement(
 							__(
-								'Jetpack and its parent company Automattic take data privacy and the GDPR very seriously. We respect the GDPR’s principles of minimizing the amount of data we collect, being transparent about what data we collect and how we use it, complying with EU law in regards to any data that is transferred to non-EU countries, and not keeping data longer than we need it for the purpose it was collected. You can read more about the data we collect, how data is used and shared, and how long data is retained in <a>our Privacy Policy</a>.',
+								"Jetpack and its parent company Automattic take data privacy and the GDPR very seriously. We respect the GDPR's principles of minimizing the amount of data we collect, being transparent about what data we collect and how we use it, complying with EU law in regards to any data that is transferred to non-EU countries, and not keeping data longer than we need it for the purpose it was collected. You can read more about the data we collect, how data is used and shared, and how long data is retained in <a>our Privacy Policy</a>.",
```

Comment:
The quote changes in this PR are unrelated. But in each case, we had the strong surrounded by ' but also using ' within the string as an apostrophe. Updated to double quotes around string to resolve lint errors.